tag_processor_plugin/tag_processor_plugin.go

```
@@ -637,3 +695,303 @@ func (p *TagProcessor) constructUMHTopic(msg *service.Message) (string, error) {
 func (p *TagProcessor) Close(ctx context.Context) error {
 	return nil
 }
+
+// processConditionForMessage evaluates a single condition for a message using VM pool
+func (p *TagProcessor) processConditionForMessage(condition ConditionConfig, msg *service.Message) (*service.Message, bool, error) {
```

golangci-lint (1.64.8)

func `(*TagProcessor).processConditionForMessage` is unused

(unused)